test/test_db/test_db.py

```diff
@@ -96,8 +96,8 @@ def test_function_execute_batch(self):
         indicator_group_id = row[0]
 
         # Insert test indicator
-        insert_indicator_query = f'''INSERT INTO base.indicator (name, flag_active, indicator_type_id, indicator_group_id)
-        VALUES ('{test_case_name}', true, 1, {indicator_group_id});'''
+        insert_indicator_query = f'''INSERT INTO base.indicator (name, flag_active, indicator_type_id, indicator_group_id, user_group)
+        VALUES ('{test_case_name}', true, 1, {indicator_group_id}, \'test_group\');'''
```

Comment:
Escaping is not required here as we use '''
This applies for the whole file (at least 5 more occurences)

Reply:
removed the escaping